src/QMCDrivers/DMC/WalkerControlMPI.cpp

```diff
@@ -196,7 +211,7 @@ int WalkerControlMPI::branch(int iter, MCPopulation& pop, FullPrecRealType trigg
 void WalkerControlMPI::determineNewWalkerPopulation(int Cur_pop,
```

Comment:
Is it a good idea to move functions added for batched driver to a separate WalkerControlMPIPopulation.cpp file?
WalkerControlMPI.cpp can be quite crowded already.

Reply:
I'd rather not double files but instead mark legacy functions (preferably [[deprecate]] them so new code stops being written using them)